src/app/pages/products/components/product-dashboard/product-dashboard.component.ts

```diff
@@ -25,6 +27,9 @@ export class ProductDashboardComponent implements OnInit {
   async ngOnInit(): Promise<void> {
     await this.productFacadeService.initProductsPage();
 
+    this.isProductListEmpty$ = this.productFacadeService.getAllProducts()
+      .pipe(map((products) => !products));
```

Comment:
I think we need to check the length property here, because `![]` and `![{}, {}]` will give us the same result here.
```suggestion
      .pipe(map((products) => !products.length));
```